crates/bevy_core/src/time/stopwatch.rs

```diff
@@ -0,0 +1,182 @@
+use bevy_reflect::{Reflect, ReflectComponent};
+
+/// A Stopwatch is a struct that track elapsed time when started.
+/// It requires a type `T` in order to be specialized for your systems and components.
+/// This specialization `T` does not take any additional space in the struct.
+///
+/// # Examples
+///
+/// ```
+/// # use bevy_time::*;
+/// let mut stopwatch = Stopwatch::<()>::new();
+/// assert_eq!(stopwatch.elapsed(), 0.0);
+/// stopwatch.tick(1.0); // tick one second
+/// assert_eq!(stopwatch.elapsed(), 1.0);
+/// stopwatch.pause();
+/// stopwatch.tick(0.5); // paused stopwatches don't tick
+/// assert_eq!(stopwatch.elapsed(), 1.0);
+/// stopwatch.reset(); // reset the stopwatch
+/// assert!(stopwatch.paused());
+/// assert_eq!(stopwatch.elapsed(), 0.0);
+/// ```
+#[derive(Debug, Clone, Reflect)]
+#[reflect(Component)]
+pub struct Stopwatch<T: Send + Sync + 'static = ()> {
```

Comment:
the ` = ()` should mean that the type can be `()` by default but I didn't get it to actually work in the short time I tried. I don't think it should be kept if it doesn't work or we don't want a default

Reply:
Yes I'm not sure of what it does either. It seems that this feature is not intended for using the type but rather implementing it.
I'm going to remove them then.